test/project-service.ts

```diff
@@ -273,6 +292,153 @@ describe("Project Service Tests", () => {
 			projectIntegrationTest.createProject(projectName).wait();
 			projectIntegrationTest.assertProject(tempFolder, projectName, options.appid).wait();
 		});
+
+		describe("project name validation tests", () => {
+			let validProjectName = "valid";
+			let invalidProjectName = "1invalid";
+
+			beforeEach(() => {
+				hasPromptedForString = false;
+			})
+
+			afterEach(() => {
+				helpers.isInteractive = originalIsInteractive;
+			});
+
+			it("creates project when is interactive and incorrect name is specified and the --force option is set", () => {
+				let projectIntegrationTest = new ProjectIntegrationTest();
+				let tempFolder = temp.mkdirSync("project");
+				let projectName = invalidProjectName;
+				helpers.isInteractive = () => { return true; };
```

Comment:
```TypeScript
helpers.isInteractive = () => true;
```
Also you can move this line in the beforeEach statement. You are using it in all of your tests (you will have to keep it only in the last test, where the passed value is `false`